credentialz/credentialz.proto

```diff
@@ -787,3 +797,17 @@ message GetPublicKeysResponse {
   // Return all currently configured keys on host.
   repeated PublicKey public_keys = 1;
 }
+
+message AddAccountRequest {
+  // The system role account name. This account must not already exist.
+  string account = 1;
```

Comment:
Some implementations need a bare minimum password to create an account, would suggest add `Password` which can be optional.

We might also want to clarify if `Password` is not specified we want to create an account with,
1. Disabled password (prevent authentication via password) OR
2. No password (password not required for authentication)

Reply:
(picking long past old PR)

Either specifying in the proto/docs:
  "If a passwd is required, but not provided, an error is returned"
or:
   "where a password is required and not provided the device will provide an unusable password"
     (like `username:*:userid:groupid:comment:shell` on standard unix-like platforms)

seems ok.